setup.cfg

```diff
@@ -41,6 +41,12 @@ install_requires =
     PyGithub ~= 2.0
     ruamel.yaml ~= 0.15
 
+[options.extras_require]
+test =
+    mypy
+    pytest
+    pytest-cov
```

Comment:
This isn't used by the test environment.

Reply:
@yarikoptic `pytest-cov` is still unused.

Also, I would prefer it if, rather than defining a `test` extra, the test dependencies were instead listed in `tox.ini`.

Reply:
why (what advantage) to list in a tool specific `tox.ini` making it harder to setup environment for any other tool/direct invocation of pytest?  I would really prefer to concentrate dependencies in a single location.

Reply:
I generally do not support any other tools than tox nor directly invoking pytest.

Reply:
I typically invoke pytest directly, and testing within debian and most often conda packages is done directly via pytest.  it is odd to require some additional utility whenever pytest framework is already sufficient.